jobs/config.json

```diff
@@ -10366,8 +10366,9 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.8",
```

Comment:
Since this is the test for upgrading from 1.6 to 1.7, should this be `ci/latest-1.7`?

Reply:
It is 1.8 because the needed test code was not introduced until 1.8. 

The extract stuff is pretty confusing. See the background section of my first post to see what I learned about what these flags do and how to use them.

Reply:
So we basically run e2e.test of v1.8?

Reply:
They way it is configured, we run all tests out of v1.8.

Reply:
Got it. Does that also mean that we'll be running 1.8 Conformance tests against the 1.7 control plane? Have you gotten a successful run of this job locally?

Reply:
I think it would be best to hold off on this part of the change until we can backport k8s anywhere upgrade support to the 1.7 e2e tests. The change for 1.7 to 1.8 looks fine, and we should get that in.